content/1.7/back-office/international/taxes/taxes-listing.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Taxes Listing page
+weight: 1
+---
+# IMPROVE > International > Taxes
+
+![Taxes](static/img/international-taxes.png)
+
+This page stores all the available Tax titles in Prestashop application.
+
+## Top elements
+
+- **Taxes** - page title.
+- **Add new tax CTA button** - redirects to tax creating and editing page.
+- **Help** - a side-bar with the helpful information, how to navigate in this page.
+- **Tabs: Taxes and Tax Rules** - navigation tabs for taxes and tax rules.
+
+## Middle elements - Tax table
+
+- **Taxes** - table title with the total taxes in the list.
+- **Numeric value** - total sum of shopping taxes identified.
+- **Export icon** - when clicked, a file is generated with all **displayed** taxes from the data list in CSV format.
+- **Refresh** - once clicked, the whole page reloads to display the latest data.
+
```

Comment:
- **Wheel icon** - when clicked, these actions are available: 
    - **Export icon** - when clicked, a file is generated with all **displayed** taxes from the data list in CSV format.
    - **Refresh** - once clicked, the whole page reloads to display the latest data.

Reply:
@camilleganzin could we specify as Gear icon? It is more like that, I guess. And there are 3 values in dropdown on PS1780 
<img width="1225" alt="image" src="https://user-images.githubusercontent.com/29095235/166876527-93312c8b-b193-456b-abf3-e20d943b4679.png">

So, what could be better? 🙂

Reply:
@camilleganzin @marwachelly hey, maybe you have insights how I should update correctly? 🙂 Thanks.